src/azure-cli/azure/cli/command_modules/backup/custom_base.py

```diff
@@ -197,6 +197,8 @@ def list_protectable_items(cmd, client, resource_group_name, vault_name, workloa
             container = show_container(cmd, container_client, container_name, resource_group_name, vault_name,
                                        "AzureWorkload")
             custom_help.validate_container(container)
+            if isinstance(container, list):
+                raise CLIError("Multiple containers with same Friendly Name found. Please give native names instead.")
```

Comment:
Please give the option name for native names in the error message to make it more clear for users. @houk-ms do we have a more detailed error type for this case?

Reply:
There is a similar error raised on line 402 of this file. I just replicated that error.

Reply:
For the new error handling cases, `CLIError` is deprecated. Check the [error handling guideline here](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md)

For this case, we can use `ValidationError`